RELEASES.md

```diff
@@ -64,7 +64,7 @@ Releases usually follow a few steps:
     * [ ] merge multi-commit PRs (so each line has a `(#num)` suffix)
     * [ ] drop hash and indent, `:'<,'> s/^\w*  /^I* /`
   * [ ] a commit bumping `./specs-go/version.go` to next version and empty the `VersionDev` variable
-  * [ ] a commit adding back the "-dev" to `VersionDev`
+  * [ ] a commit adding back the "+dev" to `VersionDev`
```

Comment:
Could you also update https://github.com/opencontainers/runtime-spec/blob/5bc62f1b8b96b00772677c784ab1f8e4bc47ebdc/specs-go/version.go#L14

Reply:
I wasn't sure whether it'd be a good idea to change the versioning for a `-dev` "release" we've already done. I think it'd be okay if we just did this for future releases.